lit-cli/client.py

```diff
@@ -0,0 +1,149 @@
+import click
+import os, shutil
+import numpy as np
+
+
```

Comment:
Documentation for all methods.  Need tests for all the different options(ONNX/MXNet Model files/MAR), I see no tests anywhere.